styleguide/Views/styleguide-grid.blade.php

```diff
@@ -0,0 +1,28 @@
+@extends('components.content-area')
```

Comment:
Thinking this should pick up the main `resources/views/grid.blade.php` instead of being a replica of the code.